docs/pt_BR/source/zip_code.rst

```diff
@@ -0,0 +1,140 @@
+Busca de endereço
+=================
+
+A busca de endereço consite em passar como parâmetro um cep válido e o pacote
+executar o serviço consultaCEP no webservice dos correios.
+
+Começando
+---------
+
+Os pré requisitos para executar este serviço são.
+
+- Ter um usuário e senha válidos
+
+- Ter o pacote instalado.
+
+- Um cep válido
+
+
+Faça os imports do client, model ZipCode e exceptions
+
+.. code-block:: python
+
+    from correios.models.address import ZipCode
+    from correios.exceptions import InvalidZipCodeError, InvalidAddressesError
+
+    try:
+        from correios import client as correios
+    except ImportError:
+        correios = None
+
+
+Realize a autenticação no webservice setando o ambiente
+
+.. code-block:: python
+
+    client = correios.Correios(
+        username='sigep-test', password='sigep-test',
+        environment=correios.Correios.TEST
+    )
+
+
+Após autenticado execute o método para buscar o cep
+
+.. code-block:: python
+
+    zip_address = client.find_zipcode(ZipCode(zip_code))
+
+Script completo
```

Comment:
this snippet doesn't seem right here in the docs. IMHO it should bee placed on an `examples/` folder